development/secrets-rotator/terraform/modules/service-account-keys-cleaner/variables.tf

```diff
@@ -0,0 +1,64 @@
+variable "project" {
+  type = object({
+    id     = string
+    number = number
+  })
+  description = "Google Cloud project ID to deploy the service account keys cleaner to."
+}
+
+variable "region" {
+  type        = string
+  description = "Google Cloud location to deploy the service account keys cleaner to."
```

Comment:
If it changed the name, the description should also reflect that this should be a region.